buildpipeline/pipeline.json

```diff
@@ -126,6 +126,7 @@
           "Parameters": {
             "PB_AdditionalBuildArguments": "/p:PortableBuild=true /p:SkipTests=true",
             "PB_TargetArchitecture": "arm",
+            "PB_DistroRid": "win8-arm",
```

Comment:
When are we ever building non-portable for windows in our official builds?  I think this should always be `win-arm`.